python/ql/src/Expressions/IsComparisons.qll

```diff
@@ -1,12 +1,16 @@
+/** INTERNAL - Helper predicates for queries concerning comparison of objects using `is`. */
```

Comment:
Perhaps simpler as:

```suggestion
/** INTERNAL - Helper predicates used by queries that test the comparison of objects using `is`. */
```

Reply:
I think _concerning_ is clearer here, since a comparison using `is` can be considered a test. I am 100% clear on what the first one means, whereas I have to think twice when reading the version using `that test the`.

Reply:
Thanks - great point. For me "concerning" seemed a little vague. May be:

```suggestion
/** INTERNAL - Helper predicates for queries that inspect the comparison of objects using `is`. */
```
?
If this doesn't work either, then you should probably just resolve my comment.